docs/operator-docs/bulk-operations.md

```diff
@@ -0,0 +1,20 @@
+---
+id: bulk-operations
+title: Bulk Operations
+hide_title: true
+---
+
+# Bulk Operations
```

Comment:
It would be nice to have a screenshot at least of the operations page on the dashboard